driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/pread64.bpf.c

```diff
@@ -71,16 +71,16 @@ int BPF_PROG(pread64_x,
 		/* We read the minimum between `snaplen` and what we really
 		 * have in the buffer.
 		 */
-		unsigned long bytes_to_read = maps__get_snaplen();
-
-		if(bytes_to_read > ret)
+		u16 snaplen = maps__get_snaplen();
+		apply_dynamic_snaplen(regs, &snaplen, false);
+		if(snaplen > ret)
 		{
-			bytes_to_read = ret;
+			snaplen = ret;
 		}
```

Comment:
now all the syscalls involved in the dynamic snaplen logic follow this pattern!
Please note that the dynamic snaplen is disabled by default so this `apply_dynamic_snaplen` method should be a no-op